Userland/Libraries/LibELF/DynamicLoader.cpp

```diff
@@ -209,8 +209,9 @@ void DynamicLoader::do_main_relocations()
 {
     do_relr_relocations();
 
```

Comment:
Commit title nitpick: Do you have the metrics on hand how many symbol lookups we did originally? It sounds slightly "lopsided" if we have an absolute count of how many lookups we eliminated, but not the "out of X lookups" part.

Reply:
I changed my testing code to only care about global and weak lookups, as only they require calculating symbol hashes and comparing names. Of the 480'752 such lookups via `DynamicLoader::lookup_symbol`, 156'282 can be eliminated this way.